src/util.cpp

```diff
@@ -682,6 +682,15 @@ void ArgsManager::ParseParameters(int argc, const char* const argv[])
             m_override_args[key].push_back(val);
         }
     }
+
+    // we do not allow -includeconf from command line, so we clear it here
+    auto it = m_override_args.find("-includeconf");
```

Comment:
This should be a loop, otherwise only the first `-includeconf` is removed from `m_override_args`. Also update comment above accordingly.

Reply:
Right. Either log all or nothing?

Reply:
Makes sense. Added loop over vector.